### ISSUE TITLE
Removing old Mario-style music from TSC

### ISSUE THREAD

**Issue by Quintus** [Secretchronicles/TSC#97]
We have some music in SMC that still resembles original Mario too much. It should be removed, but for this we first need to have a list of which music actually _is_ too Mario-like.

Valete,
Quintus

**Comment by datahead8888** [comment]
If we do not find a direct replacement for land/land_5.ogg, we will need to change the default music for a new level to something else as part of our music-reassignment efforts.  This song is a remix of the Bob-Omb Battlefield music in Super Mario 64.

**Comment by datahead8888** [comment]
I took a look to see where the world 1 map music came from.  It's "Saw-Belly Bacon" from http://derekaudette.ottawaarts.com/music.php

There are a number of other songs on there probably with permissive licenses.  If we can't find original music before the release, this is another option that is better than releasing with Mario remixes.  Of course, it is preferable when we can get music from our contributors.

**Comment by Quintus** [comment]
> I imagine we'd want screenshots of the licensing for the world 1 song in the legal repository.

Will do so.

Vale,
Quintus

**Comment by Quintus** [comment]
> Will do so.

Done.

**Comment by datahead8888** [comment]
Consolidated list of what remains:
* https://raw.githubusercontent.com/Secretchronicles/TSC/release-2.0.0/tsc/data/music/land/jungle_1.ogg - Super Mario World land level music
* https://raw.githubusercontent.com/Secretchronicles/TSC/release-2.0.0/tsc/data/music/land/land_1.ogg - Super Mario World general map music
* https://raw.githubusercontent.com/Secretchronicles/TSC/release-2.0.0/tsc/data/music/land/land_2.ogg - Super Mario World Vanilla Dome music
* https://raw.githubusercontent.com/Secretchronicles/TSC/release-2.0.0/tsc/data/music/land/land_5.ogg - Super Mario 64 Bob-Omb Battlefield music (this is also our default music for new levels right now)
* https://raw.githubusercontent.com/Secretchronicles/TSC/release-2.0.0/tsc/data/music/land/water_1.ogg - Super Mario Bros. Invincibility music
* https://raw.githubusercontent.com/Secretchronicles/TSC/release-2.0.0/tsc/data/sounds/player/dead.ogg - Super Mario World player dying music

If anyone thinks I missed one, let me know.  These links to the raw files will be useful in finding musicians to help us knock this blocker task out.

**Comment by datahead8888** [comment]
Our first musician recruiting post, requesting replacements for the songs listed above:
http://opengameart.org/forumtopic/the-secret-chronicles-of-dr-m-0

**Comment by Quintus** [comment]
Thanks for your effort!

Vale,
Quintus

**Comment by datahead8888** [comment]
3 people have replied so far to the game jolt posting at http://gamejolt.com/community/forums/topics/secret-chronicles-urgent-need-for-music/5645/

Silensir asked about helping.  His music can be listened to at:
https://soundcloud.com/silensir

Brett Cahill (InContrast) is also interested in helping.  His music can be listened to at:
https://soundcloud.com/in-contrast

Qwak asked about helping.  He posted a link to his music at:
https://soundcloud.com/qwak

Be on the look out for songs in these libraries of music that might work well Mario-music-replacements or as new music for our new goals.  We could then ask the authors if they might give us permission to use them.

**Comment by datahead8888** [comment]
Patrick Friel just responded and said he is interested in composing for Secret Chronicles.
His sample music can be heard at:
https://frielmusic.wordpress.com

**Comment by sydneyjd** [comment]
My opinion on Silensir's music:

The rising army might work, but it would be more industrial setting. Something with pipes.
The last 3 songs are too fast for the game.
I cannot think of a level that would fit Cassette.
Belt has too many "cross sounds" so it doesn't sound very good IMO.
Sweets would work i think but i don't like that it has actual talking in it.
Make me does not seem to fit the game.

**Comment by sydneyjd** [comment]
Brett Hill's songs:

A Pirate's tale: too dynamic- would fit a movie, not a game.
Selfless defender: Too orchestral.
Stagecraft: too movie'ish again.
Pangeo: too quiet.
The Wanderer: Movie-ish again.
Aftermath: Movie-ish.
Pangeo-Spring-Autum: No.
Pangeo-Winter: would fit a winter level. A little too slow many, but would give us options.
Pangeo-underwater: Fits underwater well. Reminds me of STK's underwater level music.
A lost child: No
45 Seconds or die: No, to movie-ish again.
Pangeo-Space: Underwater feeling again.
Pangeo-Junkyard: Maybe, too jazy for my liking.
Visiting the market: No.Sounds Italian or something.
Memories: No.
Pangeo-Toxic Biome: Should make good music. (maybe)? It reminds me slightly of a supertux track.
Pange-Desert: Eh, no, sounds too much like a practice drummer.
Pangeo-introduction: No
The end of everything: No
Pokemon RBY: No.
Steady tension: Underground level?
Icecream drama: No
Venture into the forest: No
Goblins march to war: No
Tainted Chi: No
Path of blight: No
Adieu: No 
Scourage: maybe, probably not. Drummer odd again.
The weather trio: No.
Evil Dr Podge: Probably not. it isn't TSC style.
Ascend: No
Moonlight: No
The river of light: No
VN piano: no
Cinamatic entrance: Industrial maybe? Or desert?

**Comment by sydneyjd** [comment]
Ashbury's songs:

I did not really find any i liked so i will have to leave it up to @datahead8888's judgement.

**Comment by datahead8888** [comment]
Here's an out of the box suggestion --> Replace land5.ogg with the Planet Plant music by Johan Brodd.  It's bright and cherry as needed, and we're already using it in the first level.  Thus it would become the new default music.  Does anyone have any concerns with this?  It would solve a problem in a whole lot of levels that use the default music very quickly.

**Comment by sydneyjd** [comment]
I'm OK with it. I think it fits well.

-Sydney

**Comment by datahead8888** [comment]
After adding Ashuburry's songs, if we don't have more music replacements, it's probably time to start talking about just moving existing music around or outright removing some songs in the worst case (and reassigning official levels to undeleted songs).  Otherwise this task alone is what will delay or release.

The dying song is a tough one.  If the musicians don't replace it, we will have to hunt for one off a website with a permissive website.  Our existing song is a derivative work of Nintendo, which could be viewed as copyright infringement.  I don't want to do it (just like I'm not eager to remove the turtle boss), but the other option is *not to have dying music* for release 2.0.

**Comment by sydneyjd** [comment]
What if we don't do a dying song, what if we do some hurt sound affect 
such as this but different? 
https://cloud.openmailbox.org/public.php?service=files&t=6b2d1cac05f32d760c06c50b829bac78&download
That is from Supertux, they use it when Tux dies.

-Sydney

**Comment by datahead8888** [comment]
> What if we don't do a dying song, what if we do some hurt sound affect

Yep, this is the other major option.  I'd still be interested in having music if possible.

It's probably time to ping silensir again about a license for his dying music...

**Comment by sydneyjd** [comment]
Yes, i would prefer music also. Dying music shouldn’t be too hard to do.

**Comment by datahead8888** [comment]
I pinged silensir on the existing dying music (we need a license).
I might ask Patrick Friel about whether he can compose it again, too.

Otherwise, we might consider borrowing a sound effect for this from Super Tux as Sydney suggested.  If someone can find a different sound effect, it will let us be unique on this.

**Comment by Quintus** [comment]
> Otherwise, we might consider borrowing a sound effect for this from Super Tux as Sydney suggested.

What licensing does Supertux use for their assets?

Vale,
Quintus

**Comment by sydneyjd** [comment]
Ok, this is the Info i got.

" 10:10 <+sydney> giby: What licence is supertux stuff?
11:12 <@giby> sydney, yes they can, they just have to mention the author 
and the "from the supertux" in credits
11:13 <@giby> Supertux is GPL
11:15 <+sydney> giby: Who is the author?
11:16 <@giby> wansti I guess
11:16 <+sydney> giby: Ok. thanks.
11:16 <@giby> Look on his page, by the way he has unused music that may 
be used too "

Here is Wansti's page also. https://discarded-ideas.org/music

**Comment by datahead8888** [comment]
Sydney brought up a really good point.  He said that some of the music on Wantsi's page is not currently used in Super Tux.  This might be a "treasure trove" as giby said of music to save the day for us.

Best of all, to quote the webpage:
> All songs are licensed under a Creative Commons Attribution-Share Alike 3.0 License, unless stated otherwise. Derivatives must use the same or compatible license or, at your option, its non-commercial version (CC-by-sa-nc). For special permissions, please contact me.

**Comment by Quintus** [comment]
> does this license work for us like CC-BY does?

The SA module is a bit dangerous, because it’s a copyleft statement. Whether we can use it, depends on whether we qualify the entire game as a single work, or as a compound work (i.e. a collection of separate works). If the former, SA licenses are unusable for us because they collide with the copyleft clause of the GPLv3 for the code, because both licenses force to license all the game in GPLv3 or CC-SA. If the latter, SA is ok, because the copyleft clause does not affect entirely separate works such as the game source code in particular. My personal opinion is the latter, and that seems to correspond to what most people think given that Luiji has shown this nice list of projects that use GPL licensing and SA graphics.

CC-BY doesn’t have this problem due to the missing SA module. As far as the NC module is concerned, I’ve explained its problems to you in IRC yesterday; it’s hardly clear to say what is a “commercial” use. Just think about nonprofit organisations using it or people demanding packaging costs. Both can be seen as commercial use, and the former has even been ruled commercial by a German court recently (though the decision was revised later to not being commercial).

> Derivatives must use the same or compatible license 

Boils down to the question what a derivative is. The GPLv3 is _not_ a compatible license, as I explained above. If we follow the view that determines the source code is not a derivative, including SA work is fine. This is my opinion, so I’d be OK with including it, but I want everyone to be aware of the possibility one can view at this from a different angle.

Vale,
Quintus

**Comment by brianvanderburg2** [comment]
Does anyone have an opinion on the Free Art License:
```
4. INCORPORATION OF THE WORK
Incorporating this work into a larger work that is not subject to the Free Art License shall not challenge the rights granted by this license.
If the work can no longer be accessed apart from the larger work in which it is incorporated, then incorporation shall only be allowed under the condition that the larger work is subject either to the Free Art License or a compatible license.

5. COMPATIBILITY
A license is compatible with the Free Art License provided:
it gives the right to copy, distribute, and modify copies of the work including for commercial purposes and without any other restrictions than those required by the respect of the other compatibility criteria;
it ensures proper attribution of the work to its authors and access to previous versions of the work when possible;
it recognizes the Free Art License as compatible (reciprocity);
it requires that changes made to the work be subject to the same license or to a license which also meets these compatibility criteria.
```

**Comment by datahead8888** [comment]
I reached silensir about the licensing for his dying music.  I am going to ask him for the licensing statement.  I will have to post a screenshot, since it will come from a soundcloud PM.

**Comment by Quintus** [comment]
> Does anyone have an opinion on the Free Art License:

This statement:

> Incorporating this work into a larger work that is not subject to the Free Art License shall not challenge the rights granted by this license.

Makes it clear to me that adding the asset into a larger project works just fine.

However, @brianvanderburg2 could you please post a link to the full license text? There might be definitions at work I can’t deduce from this.

Vale,
Quintus

**Comment by brianvanderburg2** [comment]
Here it is: http://artlibre.org/licence/lal/en/
I can't seem to find a plain txt version.  When I put a license with a work, I like to have a text document of the complete license text and not just a link to a site.  In the case of something like TSC, perhaps we could have a licenses folder underneath doc with each license used.

**Comment by brianvanderburg2** [comment]
Here it is: http://artlibre.org/licence/lal/en/
I can't seem to find a plain txt version.  When I put a license with a work, I like to have a text document of the complete license text and not just a link to a site.  In the case of something like TSC, perhaps we could have a licenses folder underneath doc with each license used.

**Comment by Quintus** [comment]
Ok, I’ve read through it and revise my explanation. The Challenging statement qutoed above wants to guarantee that if the work is added into something larger, the user of the larger work is still allowed to use the original work under the terms of the FAL.

The FAL is a copyleft license, as section 2.3(4) outlines. It requires to distribute any adaptions under the FAL or a compatible license. What a compatible license is, is defined in section 5, which Brian has posted above. Judging from these criteria, the GPLv3 looks compatible.

So as a summary: Following the more relaxed interpretation that the game is not a work on its own, using FAL assets is fine anyway, because of no adaption. Following the stricter view that the game is a work on its own, a FAL asset can still be used, because the GPLv3 and the FAL are compatible copyleft licenses with special respect to section 5(5) of the FAL. That is, we can use FAL assets in any case.

Vale,
Quintus

**Comment by datahead8888** [comment]
@giby, this is our music ticket.  You can see all of our debates above on what we're looking for if you skim through it quickly.

**Comment by datahead8888** [comment]
I posted a license confirmation from silensir for the dying music in #260.  Let me know of any concerns; otherwise we can get this music committed and be done with it.

**Comment by brianvanderburg2** [comment]
@Quintus 
I agree with that interpretation.  I would also say that it would be compatible with "GPLv3 or later" if that clause is used in the game license (I don't think it is though).  Since the FAL states that, even if used as a larger work, the original work from within that larger work must still usable by others under the original license (FAL), as long as GPLv4 and later don't add any restrictions saying that this can not be done, it should be fine.  And I can't imagine GPLv4 or later doing that, because that would break even using something licensed under a BSD/zlib style license within a GPLv4 work.

**Comment by brianvanderburg2** [comment]
In fact, compatibility only seems to matter "If the work can no longer be accessed apart from the larger work in which it is incorporated ..."  That "if" seems to mean that as long as others can somehow access the work used within apart from that larger work, you can use the work as part of a larger work under any circumstances you want, including commercial-only works, or non-commercial only works.  While this doesn't apply to us, for those that it did, they could make the asset available apart from the larger work in the form of a separate download "used assets" or something.

**Comment by Quintus** [comment]
Correct, @brianvanderburg2.

>  I would also say that it would be compatible with "GPLv3 or later" if that clause is used in the game license

Quote from README:

> This program is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.

As the README was added by us: This statement was taken from the existing copyright statementy by FluXy that were on top of the source code files. So yes, we are GPLv3+, as original SMC was.

Vale,
Quintus

**Comment by Luiji** [comment]
@Quintus GNU itself explicitly states that the GNU GPL and the FAL are incompatible. https://gnu.org/licenses/license-list.html#FreeArt The "not a larger work" interpretation might work but IANAL.

**Comment by Quintus** [comment]
@Luiji That’s interesting. The page just states that GPL and FAL are incompatible without substanciating that claim. From the license text above I beg to differ from that interpretation, but my opinion might well be wrong. In that case, the situation is exactly the same as with the CC-BY-SA license, which is also incompatible with the GPL due to conflicting copyleft clauses. However, as the FAL tries to define requirements for compatible licenses and I think that the GPL satifies this criteria, I am under the impression that incorporating a FAL work into a GPL work is OK. This surprises me, see my above explanations. Clause 2.3(1) states that the copyleft requirement steps back behind a "compatible license", i.e. there’s no need to license your derivative as FAL if you license it under a compatible license instead. Compatible licenses are defined in section 5:

> A license is compatible with the Free Art License provided: it gives the right to copy

GPL does that for sure.

> distribute

Definitely yes.

> and modify copies of the work

Yes.

> including for commercial purposes

Yes.

> and without any other restrictions than those required by the respect of the other compatibility criteria

This is incompatible with **GPLv2**. However, **not** with GPLv3. GPLv3 has itself compatibility clauses that should permit to reduce its requirements to those listed already. Hence, GPLv3 is fine.

> it ensures proper attribution of the work to its authors

Yes.

> and access to previous versions of the work when possible

Yes.

> it recognizes the Free Art License as compatible (reciprocity)

For that we have to dive into GPLv3 compatibility criteria. Just wait a moment.

> it requires that changes made to the work be subject to the same license or to a license which also meets these compatibility criteria.

This is no problem as GPLv3 does exactly that.

Ok, so now for 

> it recognizes the Free Art License as compatible (reciprocity)

So then let us [dive into GPLv3](https://gnu.org/licenses/gpl-3.0). What interests us is section 7 of the GPLv3, which is completely new and has not existed in GPLv2. Quote (emphasis by me):

> “Additional permissions” are terms that supplement the terms of this License **by making exceptions from one or more of its conditions**. Additional permissions that are applicable to the entire Program shall be treated as though they were included in this License.

This means, if the criteria following are satisfied, GPLv3 accepts it.

>  a) Disclaiming warranty or limiting liability differently from the terms of sections 15 and 16 of this License

The FAL has no disclaimer. That’s fine under this condition.

> b) Requiring preservation of specified reasonable legal notices or author attributions in that material or in the Appropriate Legal Notices displayed by works containing it

Section 2.2 clause “specify to the recipient the names of the author(s) of the originals, including yours if you have modified the work” is covered by this (and for the "yours if you have modified the work" part by c) below).

> c) Prohibiting misrepresentation of the origin of that material, or requiring that modified versions of such material be marked

This clause allows the attribution conditions of section 2.3 of the FAL (“indicate that the work has been modified and, if it is possible, what kind of modifications have been made”) , which under the GPLv2 would have not been possible.

I can’t see further problems in the FAL text. Thus, it’d say that GPLv3 is fine and GPLv2 isn’t.

However, it is probably safer to follow GNU’s recommendations even though they have not given reasons for them.

Vale,
Quintus

**Comment by datahead8888** [comment]
I would suggest moving any further license discussions not directly related to music acceptance to a new ticket / forum posting.  This ticket's discussion is already long (and needs to be long).  This will make it easier to find relevant music discussion.

**Comment by Quintus** [comment]
It just occured to me that

> it recognizes the Free Art License as compatible (reciprocity)

could also mean a round-tripping requirement. That is, the target license must also allow to step back behind the FAL when certain criteria are fulfilled that don’t conflict with the FAL’s ones. In that case, GPLv3 wouldn’t be compatible neither probably, because GPL has a strong attitude towards being the One True License™. I’m not sure about that one, though. A round-tripping requirement is something I’ve never seen before.

> I would suggest moving any further license discussions not directly related to music acceptance to a new ticket / forum posting

You are probably right with this. I apologise. @brianvanderburg2, @Luiji can we further discuss this in the forums or on the ML if you want to continue on this? This is not really something for the bugtracker.

Vale,
Quintus

**Comment by brianvanderburg2** [comment]
@Quintus I agree.  I've created this post on the forum: http://forum.secretchronicles.de/topics/221

**Comment by Quintus** [comment]
Back on topic, what remains are three songs:


* https://raw.githubusercontent.com/Secretchronicles/TSC/release-2.0.0/tsc/data/music/land/jungle_1.ogg - Super Mario World land level music
* https://raw.githubusercontent.com/Secretchronicles/TSC/release-2.0.0/tsc/data/music/land/land_1.ogg - Super Mario World general map music
* https://raw.githubusercontent.com/Secretchronicles/TSC/release-2.0.0/tsc/data/music/land/water_1.ogg - Super Mario Bros. Invincibility music

@RedFT do you got a chance to work on something for TSC again? It would be especially nice to have unique replacement for the invincibility music. Also, you might want to participate in our [General Discussion](http://forum.secretchronicles.de/topics/220) with the entire TSC team on 21st.

Vale,
Quintus

**Comment by Danfun64** [comment]
https://raw.githubusercontent.com/Secretchronicles/TSC/release-2.0.0/tsc/data/sounds/player/dead.ogg - Super Mario World death music

**Comment by Danfun64** [comment]
https://raw.githubusercontent.com/Secretchronicles/TSC/devel/tsc/data/sounds/player/dead.ogg

**Comment by datahead8888** [comment]
@Danfun64 - yes, we merge release-2.0.0 into devel on a semi-regular basis.  It's usually done after some code changes go into the release-2.0.0 branch, but we haven't been doing many code changes in that branch.

**Comment by DarkAceZ** [comment]
Are we waiting for a replacement for a jungle theme? Because I think "planet_plant" could very well fit a jungle theme... (and the filename is rather obscure anyway)

**Comment by datahead8888** [comment]
We already were talking about using planet_plant for one of the Mario replacements, so this is a good direction.

We need to find replacements for the remaining ones in our Mario music list.  Are there any other new songs we can shuffle?

If we get the songs clipped from Ashburry, one of those might work for the Mario invincibility music, but we cannot count on that.

**Comment by Quintus** [comment]
@RedFT @Aakburns what’s your current state? Do you think you can provide us with some music?

Vale,
Quintus

**Comment by sydneyjd** [comment]
If we get desperate enough we do have these to fall back onto, but i don't think many fit our game well. https://discarded-ideas.org/music 
'Other Songs' are the ones NOT used in supertux.
Wansti uses the  Creative Commons Attribution-Share Alike 3.0 License though.

If you can come up with something @RedFT @Aakburns It would be very helpfull. ;)

I really don't want to have to try to make some, since i know how it would sound. jk :P

**Comment by sydneyjd** [comment]
Eh, thats Wansti's music if you didn't catch that note. sorry. ;)

**Comment by datahead8888** [comment]
Good point, Sydney.  We need to check out Wantis's music some more and see if it will work for us.  Thanks for the reminder :)

**Comment by Quintus** [comment]
As I’ve noted in IRC, @RedFT tries to submit something when he finds the time. That’d be great :-).

Apart from that, we have replaced most of the Mario musics, with three ones still remaining. I will review sydney’s music tomorrow, and if I find something useful, I’ll include it. After that I think we can remove Blocker state from this task and remove it from the 2.0.0 milestone. It should be kept open, though, as their still is this music then probably (unless sydney’s links will give us replacements for all).

**Comment by datahead8888** [comment]
Did we use planet_plant as a replacement for other music?  That is something that might knock off one song.  If necessary, we could even shuffle other songs around.

Yes, we need to look at wantsi's music and see if RedFt can add anything else as well.  We might also ask RedFT for permission to use some of the music on his newgrounds page - he has a lot of music there, and we may have reviewed some of it way above (I can't check now).

**Comment by kirbyfan64** [comment]
Ok...I'm new here, so please have pity.

I'm what you might call an *amateur* music composer, but I was wondering if there was anything I could contribute here. I finally got FL Studio to work on my Linux box and recently downloaded some cool-sounding VSTs...

**Comment by datahead8888** [comment]
@kirbyfan64 - Yes, certainly - we gladly evaluate any submissions.  We do try to make sure the submission will work for the game, but we will gladly work with you if needed to get it to where we can use it.

**Comment by kirbyfan64** [comment]
@datahead8888 Ok...so which ones exactly are still needed?

**Comment by Quintus** [comment]
@kirbyfan64 these three need a replacement: https://github.com/Secretchronicles/TSC/issues/97#issuecomment-78041751

No remixes please, you are free to make up your very own song and are encouraged to. Just try to loosely keep the atmosphere (i.e. if the song is slow, don’t make punk music :-))

Any entirely new songs are appreciated as well of course :-)

Valete,
Quintus

**Comment by kirbyfan64** [comment]
@Quintus Got that! No remixes!

**Comment by Quintus** [comment]
I have reviewed want’s site, here are my results...

* Oldschool reEducated: Too fast, to rocky, and with with vocals.
* Prophecy: nice song, but I can’t think of any use for it in our game.
* Snowball theme surprised: The song itself is nice, but this laughing in it won’t work out for us.
* Nightfall: Doesn’t work for a level or a world, but would make for a nice story music. Maybe replace `story_1` with it, or add it as `story_2`?
* VOIPatience: I refuse to add a song to our game that is used in a telephone waiting loop.
* Forging worlds: Medieval music that’d be nice for an RPG, but not for a platformer.
* Feel for the hammer: I could imagine this in a level, but given the lyrics it’s obviously not intended to.

We probably don’t want to copy the Supertux music so I haven’t looked into it. As for the note sheets... Well, I have the silent dream that one day, we’ll have a (title?) theme in the game recorded as an orchestral version rather than computer music. But this is far, far in the future. Also, that’s still Supertux music and we probably don’t want that.

Anyone else wants to comment on these songs?

Valete,
Quintus

**Comment by datahead8888** [comment]
Oldschool reeducated - it actually may be "too retro" for our current style in some places; the vocals are also out of place for our current style as Quintus said
Prophecy - It sounds like one we could use, but I don't know of a Mario song it could directly replace at the moment.  If we want dramatic map music for the new story, it could "maybe" be an option for a direct replacement.  Otherwise it would need to be used as a new song.
Snowball surprise - As Quintus said the laughing will probably not work for us.  Also, the "surprise" vocal may not work either.
Nightfall - This might work for the tutorial, as Quintus said.  It is a loss not having a tutorial song at the moment.  It reminds me a little of Naerbu's music, actually.
I was originally assuming we might use something more "adventurous" for the tutorial, but something "otherworldly" like this for the tutorial might work, too.  We can always try new songs for the tutorial later.
VOIPatience - I wouldn't say we should never have music from a telephone loop necessarily, but I don't think this song will work in a level or map.
Forging Worlds - It won't work for a Mario replacement.  As Quintus said, I can't think of a way to use this in a new level, but we can see if our needs change over time.  You never know, if we have a Shakespeare quoting villain.
Feel the Hammer - I like the song.  The Battlecry lyrics are from a tool, not a copyrighted work, correct?  If it's just a tool, there probably aren't any legal issues.  It could probably only used for a brand new song.

We might also consider emailing Wansti.  We could ask if he either would be willing to remove vocals/laughing for songs for TSC or if he'd be interested in composing some new songs.  I would not count on this for release 2.0 music, however -- it may take time to reach him.

**Comment by Quintus** [comment]
So, far we (i.e. @datahead8888, @sydneyjd and me) have in IRC come to the following suggestions:

* Replace `jungle_1` with `planet_plant`
* Replace `land_1` with @kirbyfan64’s preliminary version from [the forum](http://forum.secretchronicles.de/topics/230), while awaiting @kirbyfan64’s later changes to it.
* Replace `water_1` with the Frankie song from [the forum](http://forum.secretchronicles.de/topics/212)

@sydneyjd is currently editing the Frankie song to properly loop. If no-one objects, I’ll make the necessary changes to the repository once he is done.

Valete,
Quintus

**Comment by datahead8888** [comment]
One more suggestion we discussed (though I didn't hear @sydneyjd 's opinion on it):
* Add back the game tutorial music, which was removed earlier because it originally had Mario music.  Use wansti's Nightfall music for it.

**Comment by sydneyjd** [comment]
To tell the truth, it would work, but i don't think it fits our game style so well. To tell the truth, it sounds like something Supertux would use.

**Comment by sydneyjd** [comment]
@datahead8888 @Quintus @DarkAceZ Opinions? https://cloud.openmailbox.org/public.php?service=files&t=fa810b52fe0ba8c675f18dbe28c468c2
I redid it so it fades in and fades out a little smoother.

**Comment by datahead8888** [comment]
The looping sounds better now.  If I crank up the volume, some of the instruments seem harsh, but it could just be my own sound hardware.  Quintus, if you can double check the sound settings on the song, though, it would not be bad to double check just in case.

**Comment by Quintus** [comment]
> @datahead8888 @Quintus @DarkAceZ Opinions?

This is very nice already. It loops properly.

One thing: If you look at the amplitudes in Audacity you’ll notice it looks like the screenshot [in the wiki](http://wiki.secretchronicles.de/Music%20and%20Sounds%20Guide). VLC automatically mixes down the volume to prevent that from being a problem, but most notably the SFML music system doesn’t do that and you’d get a crackling sound. This might also be the reason for datahead’s finding on high volumes. The quality set for the OGG file is too low -- the bit depth doesn’t suffice to contain the sounds of the music, exceeding the maximum value it allows in your OGG file. You can use Audacity’s “Normalize” effect to mix the sounds down to fit into the chosen bit depth.

Other than this, I think it’s fine and could be included.

Vale,
Quintus

**Comment by sydneyjd** [comment]
Im sorry, How clumsy of me. Here is a updated version with the clipping fixed. @Quintus @datahead8888 
https://cloud.openmailbox.org/public.php?service=files&t=88cecac2e9d96761d74daeaa98c662de

**Comment by DarkAceZ** [comment]
It's a nice track, but like @sydneyjd said, I'm not sure it fits the game too well... I guess I'd have to put it to a level first to be sure.

**Comment by Quintus** [comment]
The ANT2 music and Nightfall should be saved for a subsequent release. We have enough new music in the 2.0.0 release already.

Vale,
Quintus

**Comment by Quintus** [comment]
Cf. #411 and #412.

Valete,
Quintus

**Comment by Quintus** [comment]
I am quite happy that @datahead8888 has noticed that ALL THE SONGS HAVE BEEN REPLACED. As far as I can see, we have no more Mario remixes remaining in the game. @kirbyfan64 had the last piece we needed.

If anyone still finds one, please mention it now. Otherwise I’m going to close this ticket.

Valete,
Quintus

**Comment by datahead8888** [comment]
I'd be interested in seeing the tutorial music replaced, since we removed the Mario-music version.  This is logged in task #355.  It is not, however, required for the release or even this task.  It's just something we can do if we know of a replacement, such as wantsi's music.

**Comment by Quintus** [comment]
Great. That means we can close this issue now. Btw, thanks @xet7 for fixing the level music :heart:

Valete,
Quintus
